server/cache.js

```diff
@@ -17,6 +18,11 @@ module.exports = Cache;
 
 function Cache(ecs){
   Emitter.call(this);
+  this.tasks = LRU({
+    max: 1000000,
+    maxAge: ms('1d'),
+    length: (val, key) => { val.length }
```

Comment:
Two comments here:

1. You'll want to remove the brackets. The `return` is automatic when omitted. Currently, this is returning `undefined`.
2. `val` is the response from `ecs.describeTaskDefinition` which is an object, so `val.length` would be `undefined`.

You likely want to remove this option completely as it defaults to `function(){return 1}`. The `length` option is only necessary when you want to do something besides a count of items.